pulp_rpm/app/models/repository.py

```diff
@@ -319,6 +319,7 @@ class RpmPublication(Publication):
     package_checksum_type = models.CharField(choices=CHECKSUM_CHOICES, max_length=10)
     gpgcheck = models.IntegerField(default=0, choices=GPGCHECK_CHOICES)
     repo_gpgcheck = models.IntegerField(default=0, choices=GPGCHECK_CHOICES)
+    sqlite_metadata = models.BooleanField(default=False)
```

Comment:
Could you remind me why we are adding it (and other options) as fields on the publication?
Why is it not a one-time option?
I don't have a strong opinion either way, just trying to remember why we are doing it that way.

Reply:
I don't think we ever made any decision about where to put it, so the "why" is simply that it's where I put it :)

I wouldn't object to moving it to the repository model if you think that would be better.

Reply:
+1 to keep it on the publication if we need to have it somewhere. I wondered if we there is need to save it in the DB at all.

Reply:
I had the same thought, and it doesn't *really* need to be saved in the DB, but considering that we are saving the other parameters like gpgcheck, I did that here as well.